src/components/Shared/UI/DTESelect/DTESelect.tsx

```diff
@@ -28,6 +28,21 @@ const StyledSelect = styled(Select)`
     }
   }
 `;
+
+function capitalizeWords(input: string) {
```

Comment:
try and keep your code style consistent.  In this function you are using the function keyword but in the others you are using the more modern es6 syntax, ie
`const functionName = () => { functionBody }`

The es6 would be my preferred way